src/rest_framework_dso/serializers.py

```diff
@@ -156,6 +172,23 @@ def many_init(cls, *args, **kwargs):
     def to_representation(self, instance):
         ret = super().to_representation(instance)
 
+        display_fields = self.get_fields_to_display()
+        if display_fields is not None:
+            if not (set(display_fields) <= set(ret.keys())):
+                # Some of `display_fields` are not in result.
+                invalid_fields = [f for f in display_fields if f not in ret.keys()]
```

Comment:
`f not in ret.keys()` can be `f not in ret` as dict.__contains__ checks for the key :-)